tests/python/topi/python/test_topi_transform.py

```diff
@@ -752,21 +752,38 @@ def check_device(target, dev):
 def verify_matrix_set_diag(input_shape, diagonal_shape, dtype, k=0, align="RIGHT_LEFT"):
     input = te.placeholder(shape=input_shape, name="input", dtype=dtype)
     diagonal = te.placeholder(shape=diagonal_shape, name="diagonal", dtype=dtype)
-    matrix_set_diag_result = topi.transform.matrix_set_diag(input, diagonal, k, align)
+    k1 = te.placeholder(shape=(1,), name="k1", dtype="int64")
+    k2 = te.placeholder(shape=(1,), name="k2", dtype="int64")
+    matrix_set_diag_result = topi.transform.matrix_set_diag(input, diagonal, (k1, k2), align)
+
+    k_one, k_two = None, None
+    if isinstance(k, (tuple, list)):
```

Comment:
Can you add some comments to this test? It's a little hard to follow.

Reply:
Sure. I will add some comments and update the PR.